src/Parameter/Distance.php

```diff
@@ -24,7 +24,7 @@ public function __construct(int $distance, string $unit)
         }
     }
 
-    protected static function getConstants()
+    protected static function getConstants(): array
```

Comment:
An `array` of what? Should we type that with a doc block?

Reply:
I thought it was clear from context since all the constants in the class are strings, and since this is a private method (`protected` in this commit but updated to `private` later) I didn't think it was worth the docblock. But I'll add one for clarity. 👍

Reply:
The thing is, technically it would be `mixed[]` since the class constants could have any value in theory 😄 
I'll set it to `string[]` since that's what we have in reality.

Reply:
Fixed in https://github.com/cultuurnet/search-v3/pull/43/commits/4e60a5e3d57c40a3d1b382ab2c79aefeba42a807